config/default/lint-requirements.txt.j2

```diff
@@ -1,10 +1,2 @@
-black==22.12.0
-check-manifest==0.49
-codespell==2.2.2
-flake8==6.0.0
-isort==5.11.4
 pipdeptree==2.3.3
-pyroma==4.1
-pyupgrade==3.3.1
 z3c.dependencychecker==2.10
```

Comment:
If the idea is to move most, all? 🤔 , linting/formatting to pre-commit, then we might as well add the version pins directly on `tox.ini` or? Otherwise it feels a waste to add a file only for a couple of version pins

Reply:
Done.